app/subscriber/src/features/my-reports/edit/content/stories/ContentForm.tsx

```diff
@@ -43,20 +47,47 @@ export const ContentForm: React.FC<IContentFormProps> = ({
 }) => {
   const [{ userInfo }] = useApp();
   const [{ impersonate }] = useProfileStore();
+  const [{ tonePools }] = useLookup();
 
   if (!content) return null;
 
   const userId = impersonate?.id ?? userInfo?.id ?? 0;
   const isAV = content.contentType === ContentTypeName.AudioVideo;
-  const versions = content.versions?.[userId] ?? {
-    byline: content.byline,
-    headline: content.headline,
-    summary: '',
-    body: isAV
-      ? content.isApproved && content.body
-        ? content.body
-        : content.summary
-      : content.body,
+
+  const sentiment =
+    content.versions?.[userId]?.tone !== undefined && content.versions?.[userId]?.tone != null
```

Comment:
As far as I can tell `tone` should never be `null`

Reply:
First time the tone can be null when the user did not save her/his tone and until tone is saved by the user, then we have a value.

Reply:
If it can be `null` then your interface is not correct.  I would recommend updating the interface if you want to handle null.  However, I don't know why you would want both `null` and `undefined`.
This is one of the reasons I'm not a fan of javascript.